apps/emqx_ft/test/emqx_ft_storage_fs_gc_SUITE.erl

```diff
@@ -183,74 +283,71 @@ t_gc_complete_transfers(_Config) ->
 
 t_gc_incomplete_transfers(_Config) ->
     ct:timetrap({seconds, 120}),
-    ok = set_gc_config(minimum_segments_ttl, 0),
-    ok = set_gc_config(maximum_segments_ttl, 4),
-    {local, Storage} = emqx_ft_storage:backend(),
-    Transfers = [
-        {
-            {<<"client43"/utf8>>, <<"file-🦕"/utf8>>},
-            #{name => "dog.cur", segments_ttl => 1},
-            emqx_ft_content_gen:new({?LINE, S1 = 123}, SS1 = 32)
-        },
-        {
-            {<<"client44">>, <<"file-🦖"/utf8>>},
-            #{name => "dog.ico", segments_ttl => 2},
-            emqx_ft_content_gen:new({?LINE, S2 = 456}, SS2 = 64)
-        },
-        {
-            {<<"client1337">>, <<"file-🦀"/utf8>>},
-            #{name => "dog.jpg", segments_ttl => 3000},
-            emqx_ft_content_gen:new({?LINE, S3 = 7890}, SS3 = 128)
-        },
-        {
-            {<<"client31337">>, <<"file-⏳"/utf8>>},
-            #{name => "dog.jpg"},
-            emqx_ft_content_gen:new({?LINE, S4 = 1230}, SS4 = 256)
-        }
-    ],
-    % 1. Start transfers, send all the segments but don't trigger completion.
-    _ = emqx_utils:pmap(fun(Transfer) -> start_transfer(Storage, Transfer) end, Transfers),
-    % 2. Enable periodic GC every 0.5 seconds.
-    ok = set_gc_config(interval, 500),
-    ok = emqx_ft_storage_fs_gc:reset(),
-    % 3. First we need the first transfer to be collected.
-    {ok, _} = ?block_until(
-        #{
-            ?snk_kind := garbage_collection,
-            stats := #gcstats{
-                files = Files,
-                directories = 4,
-                space = Space
-            }
-        } when Files == (?NSEGS(S1, SS1)) andalso Space > S1,
-        infinity,
-        0
-    ),
```

Comment:
The PR looks like an overkill to be honest. 🫠 I'd definitely suggest instead to just simplify test assertions: i.e. collect all the `garbage_collection` traces for 6 (?) seconds and accumulate stats in the check phase of `?check_trace(...)`. After all, what we really care about is that every incomplete transfer is collected eventually.